services/containers/service.go

```diff
@@ -80,6 +84,16 @@ func (s *Service) Create(ctx context.Context, req *api.CreateContainerRequest) (
 	var resp api.CreateContainerResponse
 
 	if err := s.withStoreUpdate(ctx, func(ctx context.Context, store containers.Store) error {
+
+		// Labels validation
+		if req.Container.Labels != nil {
+			for k, v := range req.Container.Labels {
+				if err := labels.Validate(k, v); err != nil {
+					return errors.Wrapf(err, "containers.Labels key,value validation error")
```

Comment:
since validate has a similar error `containers.labels` would be enough here for the message